pacsman/filesystem_dev_client.py

```diff
@@ -103,20 +124,38 @@ def series_for_study(self, study_id, modality_filter=None, additional_tags=None)
                 dataset.PatientPosition = getattr(dataset, 'PatientPosition', '')
                 series_id = dataset.SeriesInstanceUID
                 if series_id in series_id_to_dataset:
-                    series_id_to_dataset[series_id].NumberOfImagesInSeries += 1
```

Comment:
We can get rid of the private tag `NumberOfImagesInSeries` defined in dicom_interface.py:10 and loaded into the pydicom tag dict entirely.

Reply:
Good point. I will remove that tag.